config/sync/user.role.gp_author_user.yml

```diff
@@ -36,13 +36,49 @@ permissions:
   - 'edit own video files'
   - 'moderate content from draft to needs_review'
   - 'moderate content from needs_review to draft'
+  - 'revert application revisions'
```

Comment:
Not sure these make sense. Would a GP need to edit anything other than, say, GP or GP Practice content types? I'm not clear if this mirrors present NI Direct?

I think this is something that needs clarifying or correcting before this can be merged.

Reply:
Yep - GP authors/supers should only be able to edit GPs and GP practices